erp/mapper/gendarmerie.py

```diff
@@ -0,0 +1,140 @@
+            erp = Erp(
+                source=Erp.SOURCE_GENDARMERIE,
+                source_id=record["identifiant_public_unite"],
+                activite=activite,
+            )
+
+        self.erp = erp
+        self.populate_basic_fields(record)
+        self._retrieve_commune_ext()
+        self.populate_accessibilite(record)
+
+        return erp
+
+    def _parse_address(self, record):
+        res = record["voie"].split(" ")
+        try:
+            if res[0][0].isdigit():
+                numero = res[0]
+                if res[1] in ["bis", "ter"]:
+                    numero += " " + res[1]
+                    voie = " ".join(res[2:])
+                else:
+                    voie = " ".join(res[1:])
+            else:
+                numero = None
+                voie = record["voie"]
+        except (KeyError, IndexError):
+            numero = None
+            voie = record["voie"]
+
+        return numero, voie
+
+    def _import_coordinates(self, record):
+        "Importe les coordonnées géographiques du centre de vaccination"
+        try:
+            x, y = float(record["geocodage_x_GPS"]), float(record["geocodage_y_GPS"])
+            return Point(x, y)
+        except (KeyError, IndexError):
+            raise RuntimeError("Coordonnées géographiques manquantes ou invalides")
+
+    def populate_basic_fields(self, record):
+        try:
+            numero, voie = self._parse_address(record)
+
+            self.erp.private_contact_email = None
+            self.erp.telephone = record["telephone"]
+            self.erp.code_insee = record["code_commune_insee"]
+            self.erp.code_postal = record["code_postal"]
+            self.erp.numero = numero
+            self.erp.voie = voie
+            self.erp.geom = self._import_coordinates(record)
+            self.erp.site_internet = record["url"]
+            self.erp.nom = record["service"]
+        except (KeyError, IndexError) as err:
+            raise RuntimeError(f"Impossible d'extraire des données: {str(err)}")
+
+    def _retrieve_commune_ext(self):
+        "Assigne une commune normalisée à l'Erp en cours de génération"
+        if self.erp.code_insee:
+            commune_ext = Commune.objects.filter(code_insee=self.erp.code_insee).first()
+            if not commune_ext:
+                arrdt = arrondissements.get_by_code_insee(self.erp.code_insee)
+                if arrdt:
+                    commune_ext = Commune.objects.filter(
+                        nom__iexact=arrdt["commune"]
+                    ).first()
+        elif self.erp.code_postal:
+            commune_ext = Commune.objects.filter(
+                code_postaux__contains=[self.erp.code_postal]
+            ).first()
+        else:
+            raise RuntimeError(
+                f"Champ code_insee et code_postal nuls (commune: {self.erp.commune})"
+            )
+
+        if not commune_ext:
+            raise RuntimeError("Impossible de résoudre la commune: ")
```

Comment:
We're missing the commune here

Reply:
Fixed in https://github.com/MTES-MCT/acceslibre/pull/182/commits/6b8668c0c1ed67154503a09786a8425186118e41